commands

```diff
@@ -26,7 +26,7 @@ case "$1" in
     fi
 
     mkdir -p "$SERVICE_ROOT" || dokku_log_fail "Unable to create service directory"
-    password=$(date +%s | sha256sum | base64 | head -c 16)
+    password=$(openssl rand -hex 16)
```

Comment:
The hex generated from this is 32 chars but yes it is valid with postgres since on my plugin I was using a value of 32 instead of 16 :grin:
And no problem for the other datastores, I wanted your input/comments before proceeding :)